apps/transport/test/transport/runtime_configuration_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Transport.RuntimeConfigurationTest do
+  use ExUnit.Case, async: true
+  import Mox
+  setup :verify_on_exit!
+
+  test ":prod config" do
+    mox = Transport.SystemEnv.Mock
+
+    mox
+    |> expect(:get_env, fn "WORKER" -> "1" end)
+    |> expect(:get_env, fn "WEBSERVER" -> "1" end)
```

Comment:
C'est bête mais j'avais pas réalisé qu'on pouvait chainer les `expect`, merci

Reply:
You welcome. Oui et du coup c'est "composable" via des fonctions si besoin !